docs/extensions/drupal.md

```diff
@@ -57,6 +57,9 @@ The `DrupalExtension` accepts the following configuration options:
 | drupal_root | An absolute path to the base directory containing your Drupal installation.  This will be used to create a Twig filesystem loader internally. |
 | twig_options | An associative array of [options to pass to Twig](https://twig.symfony.com/api/2.x/Twig_Environment.html#method___construct).  Defaults to using a `cache` property of the Mannequin cache directory. |
 
+It also has the following methods to be used for configuration:
+* `addTwigPath(string $namespace, string $path)` Adds an additional path to the Twig loader, under a specific namespace.  Use this method to add additional namespaces to the loader.
+
```

Comment:
Maybe it's worth adding a note here that makes it clear that if you want components from the new namespaces to be available within Mannequin, they need to be included in the files scanned by the Finder in .mannequin.php as well. This change could be added here and in twig.md